server/src/test/java/org/elasticsearch/health/node/FetchHealthInfoCacheActionTests.java

```diff
@@ -128,15 +131,17 @@ private HealthInfoCache getTestHealthInfoCache() {
             healthInfoCache.updateNodeHealth(
                 nodeId,
                 new DiskHealthInfo(randomFrom(HealthStatus.values()), randomFrom(DiskHealthInfo.Cause.values())),
-                randomDslHealthInfo()
+                randomDslHealthInfo(),
+                randomRepoHealthInfo()
             );
         }
         return healthInfoCache;
     }
 
     public void testResponseSerialization() {
+        var healthInfo = getTestHealthInfoCache().getHealthInfo();
         FetchHealthInfoCacheAction.Response response = new FetchHealthInfoCacheAction.Response(
-            new HealthInfo(getTestHealthInfoCache().getHealthInfo().diskInfoByNode(), DataStreamLifecycleHealthInfo.NO_DSL_ERRORS)
+            new HealthInfo(healthInfo.diskInfoByNode(), healthInfo.dslHealthInfo(), healthInfo.repositoriesInfoByNode())
```

Comment:
I think here you changed the code a bit, I think `healthInfo.dslHealthInfo()` is `null` while here it was setting it to `DataStreamLifecycleHealthInfo.NO_DSL_ERRORS`, right?